docker-compose.yml

```diff
@@ -20,9 +20,6 @@ x-app: &app
     - GOOD_JOB_CLEANUP_DISCARDED_JOBS=false
     - GOOD_JOB_CLEANUP_INTERVAL_SECONDS=86400
     - GOOD_JOB_CLEANUP_PRESERVED_JOBS_BEFORE_SECONDS_AGO=604800
-    - HYKU_ADMIN_HOST=adl.test
```

Comment:
What address do we use to connect to the browser, if we're removing adl.test?

Reply:
That was answered up in the pull request message. But it needs to be added to the repos readme if that's how we have to do it.

Is there no way to override it to be repo-specific any more so we don't have to stop each container before launching a different repo?

Reply:
Rob mentioned it'll be https://adventist-knapsack.localhost.direct/

Reply:
per rob, it is now `https://admin-adventist_knapsack.localhost.direct` for the proprieter, and substitute tenant name for admin for the tenant.

We'll definitely want to override that back to adl eventually!

Reply:
if you want to still use dory, you can add this back to your .env.development file to override the new default one